### PR TITLE
New Template - Datastax Enterprise Edition

### PR DESCRIPTION
**Pull request by trentmswanson**

### CODE REVIEW

File: datastax-enterprise/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,296 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "region": {
+      "type": "string",
+      "defaultValue": "West US",
+      "metadata": {
+        "Description": "Location where resources will be provisioned"
```
Comment (singhkay) [inline datastax-enterprise/azuredeploy.json]: "description" for parameters metadata according to guidance from @johngossman

**Comment by singhkay** [comment]
merging for testing. @johngossman to still do CR

**Comment by trentmswanson** [comment]
I added one more change... I noticed we are to use "description" instead of "Description".  =) Which I am happy to see.

File: datastax-enterprise/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+      "type": "string",
+      "defaultValue": "uniqueStorageAccountName",
+      "metadata": {
+        "description": "Unique namespace for the Storage Account where the Virtual Machine's disks will be placed"
+      }
+    },
+    "dnsName": {
+      "type": "string",
+      "metadata" : {
+        "description": "DNS subname for the opserations center public IP"
```
Comment (johngossman) [inline datastax-enterprise/azuredeploy.json]: "operations" not "opserations"

**Comment by singhkay** [comment]
@trentmswanson Yep, we'll backfill all templates with this change

**Pull request merged**